src/main/java/interface_adapter/textspeech/TextSpeechViewModel.java

```diff
@@ -1,13 +1,33 @@
 package interface_adapter.textspeech;
 
+import java.beans.PropertyChangeListener;
+import java.beans.PropertyChangeSupport;
 
 public class TextSpeechViewModel {
-    private String audioContent;
-    public String getAudioContent() {
-        return audioContent;
+
+    private TextSpeechState state = new TextSpeechState();
+
+    public TextSpeechViewModel() {
+        super();
```

Comment:
Currently, because your TextSpeechViewModel is not implementing ViewModel, I'm not too sure what super() will do. Instead, I would suggest implementing ViewModel and adding an input into the constructor for the String viewName. Then, in the constructor, you could call super using the String input viewName, which is the constructor for ViewModel.